policy/README.md

```diff
@@ -293,4 +293,11 @@ The internal mechanics of ordering are up to the Policy editing and hosting soft
 
 [Top](#table-of-contents)
 
+[iana]: https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+[muni-boundary]: ../provider/README.md#municipality-boundary
+[versioning]: ../provider/README.md#versioning
```

Comment:
`versioning` should reference the shared document: https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/general-information.md#versioning